src/amo/css/inc/vars.scss

```diff
@@ -21,8 +21,12 @@ $showmorecard-gradient-color: #f4f9ff;
 // Fonts
 $default-font-family: 'Clear Sans', Helvetica, Arial, sans-serif;
 
+// Line-Height Variables
+$line-height-body: 1.5;
+$line-height-compressed: 1.2;
+$line-height-heading: 1.1;
```

Comment:
When I computed some of the ratios from the guidelines, it was close to `1.2` in most cases so I'd use `1.2` for everything else (we decided to do that in addons-blog fwiw).

Reply:
I'm not sure what you mean. The guidelines explicitly state that the line-heights are calculated as percentages of the font-size, using 150% for body fonts and 110% for heading fonts, so that's what I've used for our defaults as well. I did replace some line-heights in our files that are close to some of these, and got rid of the ridiculously specific ones, but there are a handful left that don't quite fall in the range so I kept them. I suppose we could just use `1.2` in some of those cases (which is why I came up with that third setting called "compressed"), but I tested visually and in most cases there is a visual difference so I figured we could keep some of the specific line-heights. 

One of the reasons this is a draft is because I was going to add a bunch of comments to explain the reasons for my decisions in these files.

Reply:
I hadn't seen the footnotes. I manually computed the line-height values based on the pixel values and it was `1.16`, `1.125`, `1.25`, etc. so we decided to use `1.2` everywhere. Maybe we should have used `1.1` then but the rounding thing isn't necessarily `1.1` in the end. `1.2` looks good.

I don't think we need 3 `line-height`, though. The guidelines mention 2, right? So let's have 2, whether the value is `1.1` or `1.2` (also note that headings should preferably use Metropolis, so maybe `1.1` isn't great for Inter).

Reply:
I'm not sure I am following what you are suggesting. I can change the default for headings to `1.2`, from `1.1`, based on your suggestion above, but are you suggesting we remove *all* explicit line-heights and just use `1.2` in all those cases? Some are used for UI tweaks, and if I remove them we get a visual regression, which is why I think we do need to keep a few exceptions.

Reply:
`1.1` vs `1.2` isn't very different so, yes, I would only keep one and apply it everywhere (when it shouldn't be `1.5`). At least, I'd try that to see how it would look. I get that it would slightly change some elements but would that introduce UI glitches?

Reply:
> `1.1` vs `1.2` isn't very different so, yes, I would only keep one and apply it everywhere (when it shouldn't be `1.5`). At least, I'd try that to see how it would look. I get that it would slightly change some elements but would that introduce UI glitches?

In some cases yes (one in particular I remember when making these changes), but in most cases no. I can take that approach and double check each case where I remove an explicit `line-height`.